extensions/VisualEditor/modules/ve-mw/init/ve.init.mw.Target.js

```diff
@@ -1893,3 +1950,62 @@ ve.init.mw.Target.prototype.scrollToHeading = function ( headingNode ) {
 
 	$window.scrollTop( headingNode.$element.offset().top - this.getToolbar().$element.height() );
 };
+
+/**
+ * Fire off the sanity check. Must be called before the surface is activated.
+ *
+ * To access the result, check whether #sanityCheckPromise has been resolved or rejected
+ * (it's asynchronous, so it may still be pending when you check).
+ *
+ * @method
+ * @fires sanityCheckComplete
+ */
+ve.init.mw.Target.prototype.startSanityCheck = function () {
```

Comment:
Why there was a need to port this method? There is no sanity check anymore in new VE? If there is no then maybe worth looking into their git to find out why the resigned form it - and we should do it too.